pytorch_lightning/utilities/cli.py

```diff
@@ -579,6 +586,7 @@ def add_core_arguments_to_parser(self, parser: LightningArgumentParser) -> None:
         """Adds arguments from the core classes to the parser."""
         parser.add_lightning_class_args(self.trainer_class, "trainer")
         parser.set_choices("trainer.callbacks", CALLBACK_REGISTRY.classes, is_list=True)
+        parser.set_choices("trainer.logger", tuple(_Registry.get_members(pl.loggers, pl.loggers.LightningLoggerBase)))
```

Comment:
Can we create a proper registry for logger ?

Reply:
I don't agree with that. I'ts adding bloat considering there are no logger implementations outside of the project.

I'm totally open to adding it when the community requests it, but adding it is unrelated to supporting shorthand.

Reply:
The point of adding a registry is for users to easily add any external library such as the aim logger or any other and benefit from shorthand without adding them within Lightning. IMO, this needs to be added :)